crates/solidity/testing/utils/src/config.rs

```diff
@@ -34,13 +33,9 @@ pub fn working_dir_path() -> PathBuf {
     Path::repo_path("target/benchmarks-inputs")
```

Comment:
nit:
while the `Project`/`File` abstractions exist here because they are shared between cargo/npm, the actual details of the `working_dir_path()` structure and reading/handling its contents is leaked into cargo/npm, and handled in both.
Should that be abstracted here, making `working_dir_path()`/`fetch()` private, so that cargo/npm can just e.g. `Project::fetch()` the contents, without needing to understand the underlying structure?